### PR TITLE
image/spec/v1.1.md formatting error

### LINKED ISSUE #38337: image/spec/v1.1.md formatting error
**Description**

**Steps to reproduce the issue:**
There is a formatting error in the file(image/spec/v1.1.md)

**Describe the results you received:**
There are two unnecessary blank lines.
```
        </ul>

Here is an example history section:

<pre>"history": [
  {
    "created": "2015-10-31T22:22:54.690851953Z",
    "created_by": "/bin/sh -c #(nop) ADD file:a3bc1e842b69636f9df5256c49c5374fb4eef1e281fe3f282c65fb853ee171c5 in /"
  },
  {
    "created": "2015-10-31T22:22:55.613815829Z",
    "created_by": "/bin/sh -c #(nop) CMD [\"sh\"]",
    "empty_layer": true
  }
]</pre>
    </dd>
</dl>
```

**Describe the results you expected:**
```
        </ul>
Here is an example history section:
<pre>"history": [
  {
    "created": "2015-10-31T22:22:54.690851953Z",
    "created_by": "/bin/sh -c #(nop) ADD file:a3bc1e842b69636f9df5256c49c5374fb4eef1e281fe3f282c65fb853ee171c5 in /"
  },
  {
    "created": "2015-10-31T22:22:55.613815829Z",
    "created_by": "/bin/sh -c #(nop) CMD [\"sh\"]",
    "empty_layer": true
  }
]</pre>
    </dd>
</dl>
```

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
(paste your output here)
```

**Output of `docker info`:**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

### PR DESCRIPTION
**Pull request by Bevisy**
**- What I did**
format  image/spec/v1.1.md
**- How I did it**
delete unnecessary blank lines
**- How to verify it**

**- Description for the changelog**
delete unnecessary blank lines in image/spec/v1.1.md

fixes #38337 

Signed-off-by: Bevisy Zhang <binbin36520@gmail.com>

### CODE REVIEW

File: image/spec/v1.1.md [hunk 1]
```diff
@@ -419,7 +419,6 @@ Here is an example history section:
     "empty_layer": true
   }
 ]</pre>
-    </dd>
```
Comment (thaJeztah) [inline image/spec/v1.1.md]: This `</dd>` should not be removed; the actual fix is to remove the blank lines at line 408 and 410 instead. Similar to what was done in https://github.com/moby/moby/pull/37051

```patch
diff --git a/image/spec/v1.1.md b/image/spec/v1.1.md
index de74d91a19..8d0ec892da 100644
--- a/image/spec/v1.1.md
+++ b/image/spec/v1.1.md
@@ -405,9 +405,7 @@ whitespace. It has been added to this example for clarity.
             filesystem).
           </li>
         </ul>
-
 Here is an example history section:
-
 <pre>"history": [
   {
     "created": "2015-10-31T22:22:54.690851953Z",
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks for contributing! I don't think this is the correct fix, but I left a comment with an alternative. Could you update your PR to use that alternative instead?

**Review by thaJeztah — CHANGES_REQUESTED** [review]
thanks! changes look good, but can you squash your commits?

Let me know if you need help doing so

**Comment by Bevisy** [comment]
@thaJeztah Thanks for your patience.I've already changed it .

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**